src/main/java/org/eclipse/tractusx/ssi/lib/exception/proof/SignatureVerificationFailedException.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * ******************************************************************************
+ * Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * *******************************************************************************
+ */
+
+package org.eclipse.tractusx.ssi.lib.exception.proof;
+
+import org.eclipse.tractusx.ssi.lib.exception.SSIException;
+
+/** The type Signature Verification Failed exception. */
+public class SignatureVerificationFailedException extends SSIException {
```

Comment:
Think about renaming this class to e.g. **SignatureVerificationException**. Term like "failed" / "check" should not be included in class names.